### ISSUE TITLE
uan代码部分可能存在问题

### ISSUE THREAD

**Issue by p949982022** [xiaoleimiao/UDTL_Lable_Inconsistent#3]
您好 在UAN代码的测试阶段的一行代码可能存在问题  for (each_predict, each_label,each_target_share_weight) in zip(logits, labels.cpu(), target_share_weight):  这个target_share_weight应该利用测试数据重新计算一下

**Comment by Hide927** [comment]
> 您好 在UAN代码的测试阶段的一行代码可能存在问题 for (each_predict, each_label,each_target_share_weight) in zip(logits, labels.cpu(), target_share_weight): 这个target_share_weight应该利用测试数据重新计算一下

和你有同样的问题，参考UAN的源码，测试阶段的weight应该是需要重新计算的
